src/lime/media/AudioSource.hx

```diff
@@ -36,6 +37,8 @@ class AudioSource
 
 		this.loops = loops;
 
+		pitch = 1;
+
```

Comment:
Remove the pitch and put it below the code that says `__backend = new AudioSourceBackend(this);`

Reply:
Can you paste the lines you're seeing? When I look at the function, I see them in order:

```haxe
__backend = new AudioSourceBackend(this);

if (length != null && length != 0)
{
	this.length = length;
}

this.loops = loops;

pitch = 1;
```

Reply:
I see them like this

```haxe
pitch = 1;

__backend = new AudioSourceBackend(this);

if (length != null && length != 0)....
```